pkg/client/example-runner.go

```diff
@@ -0,0 +1,287 @@
+package client
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/pkg/broker"
+	"github.com/pivotal-cf/brokerapi"
+)
+
+func RunExamples(client *Client) error {
+	return RunExamplesForService(client, "")
+}
+
+func RunExamplesForService(client *Client, serviceName string) error {
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	services := broker.GetAllServices()
+
+	for _, service := range services {
+
+		if serviceName != "" && serviceName != service.Name {
```

Comment:
serviceName != "" seems redundant assuming all of the entries withing services have a populated name field.

Reply:
Unless serviceName = "" implies you want all services. It might make sense to document that assumption in the godoc for this function.

Reply:
++